src/Bladeburner/Actions/Casualties.ts

```diff
@@ -0,0 +1,51 @@
+import { getRandomIntInclusive } from "../../utils/helpers/getRandomIntInclusive";
+import { OperationTeam } from "../Bladeburner";
+
+export type Casualties = {
+  deaths: number; // Actual roll
+  damagedSleeves: number; // Sleeves included in actual deaths
+};
+
+type CasualtyRolls = {
+  worstCase: number; // High end roll for potential deaths
+  bestCase: number; // Low end roll for potential deaths
+};
+
+export enum OperationCasualtyOutcome {
```

Comment:
This one is also arguable, but in this particular case I feel like the two-member enum is adding more extra verbosity than it is helping. Just using a boolean "isSevere" feels like it would cut out a lot of boilerplate.

Reply:
I had the same argument in mind. Just because it has two values, doesn't make it a good candidate for a boolean. I think boolean types should be used specifically for a YES / NO situation. In a LOW-HIGH situation I'd rather have LOW and HIGH.

If you insight, I can turn it specifically into 0.5 and 1 (1 | 0.5) and pass around the factor, but that's opening doors for architecture we don't need to support right now.

However, since this is mostly just data, I could stick it into Bladeburner constants as the default loss rate and not give a choice at all (just pass in success).

Which one would you prefer?